reflectable/pubspec.yaml

```diff
@@ -18,7 +19,7 @@ dependencies:
   logging: ^1.2.0
   package_config: ^2.1.0
   path: ^1.9.0
-  pub_semver: ^2.0.0
+  pub_semver: ^2.2.0
```

Comment:
I can't comment on the correct line because GitHub is dumb but all the build related DEPS here should probably be dropped now - if this package does use build_runner the DEPS should be moved to dev dependencies at a minimum.